src/api.ts

```diff
@@ -11,3 +12,22 @@ export function getMovie(query: string): Promise<MovieData | ResponseError> {
       Error: 'unexpected error',
     }));
 }
+
+export const newMovie = (movieData: MovieData): Movie => {
+  const {
+    Title,
+    Plot,
+    Poster,
+    imdbID,
+  } = movieData;
+
+  return {
+    title: Title,
+    description: Plot,
+    imgUrl: Poster === 'N/A'
+      ? 'https://via.placeholder.com/360x270.png?text=no%20preview'
+      : movieData.Poster,
```

Comment:
You're already destructuring 'Poster' from 'movieData', there's no need to access it again through 'movieData.Poster', simply use 'Poster'.